libraries/matrix/impl/src/test/kotlin/io/element/android/libraries/matrix/impl/fixtures/fakes/FakeSessionVerificationController.kt

```diff
@@ -0,0 +1,16 @@
+/*
+ * Copyright 2025 New Vector Ltd.
+ *
+ * SPDX-License-Identifier: AGPL-3.0-only OR LicenseRef-Element-Commercial
+ * Please see LICENSE files in the repository root for full details.
+ */
+
+package io.element.android.libraries.matrix.impl.fixtures.fakes
+
+import org.matrix.rustcomponents.sdk.NoPointer
+import org.matrix.rustcomponents.sdk.SessionVerificationController
+import org.matrix.rustcomponents.sdk.SessionVerificationControllerDelegate
+
+class FakeSessionVerificationController : SessionVerificationController(NoPointer) {
```

Comment:
The (not-written) convention is to name this class `FakeRustSessionVerificationController` so that we see that it's not a regular `Fake` class that we use for Unit test.
It seems that I did not follow the convention for `FakeQrCodeData`, my bad...